lib/web_ui/test/ui/filters_test.dart

```diff
@@ -121,22 +121,22 @@ Future<void> testMain() async {
 
   test('color filter as image filter', () async {
     const ui.ColorFilter colorFilter = ui.ColorFilter.mode(
-      ui.Color.fromRGBO(0, 0, 255, 128),
+      ui.Color.fromARGB(128, 0, 0, 255),
```

Comment:
This test was incorrectly using the `fromRGBO` method, so I fixed it.  The opacity component was supposed to be in [0,1.0].